packages/bee-q/src/components/option-group/bq-option-group.tsx

```diff
@@ -0,0 +1,85 @@
+import { h, Component, Host, Element } from '@stencil/core';
+
+import { isHTMLElement } from '../../shared/utils';
+
+@Component({
+  tag: 'bq-option-group',
+  styleUrl: './scss/bq-option-group.scss',
+  shadow: true,
+})
+export class BqOptionGroup {
+  // Own Properties
+  // ====================
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  // Prop lifecycle events
+  // =======================
+
+  // Events section
+  // Requires JSDocs for public API documentation
+  // ==============================================
+
+  // Component lifecycle events
+  // Ordered by their natural call order
+  // =====================================
+
+  componentDidLoad() {
+    const slotElements = this.el.shadowRoot
+      .querySelector<HTMLSlotElement>('[part="group"] > slot')
+      .assignedElements({ flatten: true })
+      .filter((elem: HTMLElement) => isHTMLElement(elem, 'bq-option')) as [HTMLBqOptionElement];
+
+    slotElements.forEach((bqOption: HTMLBqOptionElement) => bqOption?.setPaddingToOption());
```

Comment:
![CleanShot 2023-07-20 at 10 17 22](https://github.com/Endava/bee-q/assets/328492/ee4863b9-1abf-4839-a413-79240159783e)

Can you please provide some comments explaining what's the intention here?

Reply:
Will do! But this will be refactored soon to get simpler than that.
What it does it sets the padding to the inner option within a group
![image](https://github.com/Endava/bee-q/assets/18229977/7b2b535c-0c20-468b-9c4a-8e2436f96123)

Reply:
Maybe there's no need to use JS for that, you can probably add the style from the parent CSS by targeting the shadow part of the children (just thinking out loud here 🤔 )